### ISSUE TITLE
[Settings]: Keyboard user is not able to switch between radio button using arrow/tab key throughout the app.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#11159]
### Windows Terminal version (or Windows build number)

1.10.2383.0

### Other Software

**Test Environment:**
OS: Windows 11 Version 21H2 (OS Build 22000.168)
Screen Reader: Narrator
Tool: Accessibility Insight for Windows

### Steps to reproduce

**Repro Steps:**
Open Windows Terminal.
Open Settings page using "Ctrl + ',' ".
Verify that Keyboard user can switch between radio button using arrow/tab key throughout the app or not

**User Experience: -> Priority 1**
Keyboard users are impacted here as they will be unable to use different functionality as unable to switch between radio buttons.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/keyboard

**Attachments:**
[Keyboard user is not able to switch between radio button using arrowtab key throughout the app..zip](https://github.com/microsoft/terminal/files/7116197/Keyboard.user.is.not.able.to.switch.between.radio.button.using.arrowtab.key.throughout.the.app.zip)

### Expected Behavior

Keyboard user should be able to switch between radio button using arrow key throughout the app.

### Actual Behavior

Keyboard user is not able to switch between radio button using arrow/tab key throughout the app.

**Observation:** In Scan mode, Screen Reader user can switch between radio buttons.

**Comment by zadjii-msft** [comment]
@v-rpundir We're already tracking this in #9566, which is tracking the upstream issue in https://github.com/microsoft/microsoft-ui-xaml/issues/3156. We can't fix this ourselves, and will need to wait for WinUI to author a fix. Can we close this one?

**Comment by v-rpundir** [comment]
Closing this bug as External as we are tracking this bug here https://github.com/microsoft/microsoft-ui-xaml/issues/3156

**Comment by v-rpundir** [comment]
**Test Environment:**
**OS:** Windows 11 Version 21H2 (OS Build 22598.200)
**Application: Window Terminal Preview 
App Version: 1.13.10984.0**
**Screen Reader:** Narrator

**Comment by zadjii-msft** [comment]
@v-rpundir I'm pretty sure we got rid of all the radio buttons in the Terminal. #12833 hasn't shipped to a release version yet, but 1.14 won't have any more radio buttons to worry about the upstream bug any longer.

**Comment by v-rpundir** [comment]
Ok @zadjii-msft Will close this bug after checking on 1.14+

**Comment by DHowett** [comment]
@v-rpundir this fix is now available in version **1.13** in the Windows Insiders program, or for download on this page: https://github.com/microsoft/terminal/releases/tag/v1.13.11431.0

**Comment by v-rpundir** [comment]
Closing as fixed as not repro now.
